ros-dev-tools/README.md

```diff
@@ -0,0 +1,11 @@
+# ROS Dev Tools
+
+`ros-dev-tools` contains packages that are of general use to ROS developers but which are not dependencies of any particular packages within a ROS distribution.
+
+### Packages included in this variant
+
+* ros-build-essential
+* [rosdep](https://github.com/ros-infrastructure/rosdep)
+* [colcon-common-extensions](https://github.com/colcon/colcon-common-extensions)
+* [bloom](https://github.com/ros-infrastructure/bloom)
+* [vcs](https://github.com/dirk-thomas/vcstool)
```

Comment:
Nit: While the command-line tool is called `vcs`, the project name and package name is `vcstool`, so I'd suggest that here:

```suggestion
* [vcstool](https://github.com/dirk-thomas/vcstool)
```